core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -2262,11 +2262,12 @@ class KafkaApisTest {
       Optional.empty())).asJava
     val fetchMetadata = new JFetchMetadata(0, 0)
     val fetchContext = new FullFetchContext(time, new FetchSessionCache(1000, 100),
-      fetchMetadata, fetchData, false, metadataCache.topicNamesToIds(), false)
+      fetchMetadata, fetchData, false, metadataCache.topicNamesToIds(), false, "my-client-id")
```

Comment:
Not that it matters but should we use something like `principal=my-principal`?
Same below too.